e2e/testcafe-devextreme/tests/accessibility/floatingActionButton.ts

```diff
@@ -0,0 +1,27 @@
+import { Properties } from 'devextreme/ui/speed_dial_action.d';
+import url from '../../helpers/getPageUrl';
+import { testAccessibility, Configuration } from '../../helpers/accessibility/test';
+import { Options } from '../../helpers/generateOptionMatrix';
+
+fixture.disablePageReloads`Accessibility`
+  .page(url(__dirname, '../container.html'));
+
+const options: Options<Properties> = {
+  visible: [true, false],
```

Comment:
Check this comment, please: [https://github.com/DevExpress/DevExtreme/pull/27317#discussion_r1592029176].
Do you agree with it?

Reply:
I agree.